resources/istio/templates/job-namespace-label.yaml

```diff
@@ -25,10 +25,23 @@ spec:
           - /bin/bash
           - -c
           - |
-            set -e
+            set +e
             {{- range .Values.kyma.namespaces2Label }}
-            echo "---> Setting label to {{ . }}"
-            kubectl label namespace {{ . | quote }} "istio-injection=disabled" --overwrite
+            if RESULT=$(kubectl get namespace {{ . }} 2>&1) ; then
+              echo "---> Setting label to {{ . }}"
+              kubectl label namespace {{ . | quote }} "istio-injection=disabled" --overwrite
```

Comment:
Can we improve this a little bit? I personally find mixing bash with Helm templates a terrible idea. In this way you can't actually run/test/lint the script locally. I prefer the "old-school" approach: Have a normal bash script in file, and invoke it with paramers. Then you can do both: run/test it locally and invoke it in the job, rendering it's parameters using templates, instead of "injecting" them directly into script's statements.